src/client/gameui.cpp

```diff
@@ -128,7 +128,7 @@ void GameUI::update(const RunStats &stats, Client *client, MapDrawControl *draw_
 			<< ", " << (player_position.Z / BS)
 			<< "), yaw: " << (wrapDegrees_0_360(cam.camera_yaw)) << "° "
 			<< yawToDirectionString(cam.camera_yaw)
-			<< ", pitch: " << (wrapDegrees_180(cam.camera_pitch)) << "°"
+			<< ", pitch: " << -(wrapDegrees_180(cam.camera_pitch)) << "°"
```

Comment:
Wouldn't it be better to do the following?
```suggestion
			<< ", pitch: " << (-wrapDegrees_180(cam.camera_pitch)) << "°"
```

Reply:
I agree, i'm not sure if the outer brackets are needed, but it's clear they are meant to contain the value, so the negative sign is best inside.
+1 if done.

Reply:
Done